app/home/templates/login.tpl.jade

```diff
@@ -0,0 +1,19 @@
+div(layout="row", layout-align="center center")
+  form.login(name="loginForm" flex="35", layout="column")
+    div(layout="row", layout-align="center center")
+      md-input-container(flex="60").md-block(flex-gt-sm)
+       label(for="username")
+         <i class="fa fa-users"></i> User
```

Comment:
Because i've tried with Jade and Font Awesome didn't work as expected, at least for me.